.storybook/webpack.config.js

```diff
@@ -1,6 +1,9 @@
 let babel = require('./babel.config.js');
 let config = require('./../.config/webpack/base.js')({
     babel,
+    ts: {
+        transpileOnly: true
+    },
```

Comment:
`@storybook/react` now uses a library called `emotions` which fucks up all `css` component props expected type def in TypeScript. We use typescript, it trips us up. Removing type checking for tests like already done in Cypress.